lib/constants.ts

```diff
@@ -33,7 +33,7 @@ export const NOTEBOOK_PARAMETERS_FILE_NAME = 'notebook-params.json';
 export const PERMISSIONS_BOUNDARY_POLICY_NAME = '';
 
 // This could be something like Admin or a dedicated role for KMS Key Management
-export const KEY_MANAGER_ROLE_NAME = '';
+export const KEY_MANAGER_ROLE_NAME = 'Admin';
```

Comment:
do we always want to use admin here? do all external accounts even have a role named Admin?

Reply:
Thanks forgot to revert; will correct in next commit